Resources/Prototypes/_Goobstation/Entities/Objects/Specific/Mech/Weapons/Melee/debug.yml

```
@@ -0,0 +1,18 @@
+- type: entity
+  id: WeaponMechDebugMelle
+  parent: [ BaseMechWeaponMelee, DebugMechEquipment ] # Debug equipment does have all whitelist tags.
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)